demisto_sdk/commands/openapi_codegen/base_code.py

```diff
@@ -0,0 +1,91 @@
+base_argument = "$SARGNAME$ = $ARGTYPE$args.get('$DARGNAME$'"
+base_params = """params = assign_params($PARAMS$)"""
+base_data = """data = assign_params($DATAOBJ$)"""
+base_props = """assign_params($PROPS$)"""
+base_headers = """headers=headers"""
+base_header = """headers[$HEADERKEY$] = $HEADERVALUE$"""
+base_list_functions = "		'$FUNCTIONNAME$': $FUNCTIONCOMMAND$,"
+base_credentials = """
+    username = params['credentials']['identifier']
+    password = params['credentials']['password']
+"""
+base_basic_auth = """(username, password)"""
+base_token = """headers['Authorization'] = [f'Bearer {params["api_key"]}']"""
+base_function = """def $FUNCTIONNAME$_command(client, args):
+    $ARGUMENTS$
+
+    response = client.$FUNCTIONNAME$_request($REQARGS$)
+    command_results = CommandResults(
+        outputs_prefix='$CONTEXTNAME$$CONTEXTPATH$',
+        outputs_key_field='$UNIQUEKEY$',
+        outputs=$OUTPUTS$,
+        raw_response=response
+    )
+
+    return command_results
+
+"""
+base_request_function = """
+    def $FUNCTIONNAME$_request(self$REQARGS$):
+        $PARAMETERS$
+        $DATA$
+
+        headers = self._headers
+        $HEADERSOBJ$
+
+        response = self._http_request('$METHOD$', $PATH$$NEWPARAMS$$NEWDATA$, headers=headers)
+
+        return response
+
+"""
+base_client = """class Client(BaseClient):
+    def __init__(self, server_url, verify, proxy, headers, auth):

```

Comment:
why is init needed?

Reply:
why not?

Reply:
because all it does is call `super`, that will be done anyway when initializing the client

Reply:
but `super` is called within the `init`